gcp/table_gcp_kubernetes_node_pool.go

```diff
@@ -0,0 +1,240 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/container/v1"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpKubernetesNodePool(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_kubernetes_node_pool",
+		Description: "GCP Kubernetes Node Pool",
+		List: &plugin.ListConfig{
+			Hydrate:       listKubernetesNodePools,
+			ParentHydrate: listKubernetesClusters,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "location", "cluster_name"}),
+			Hydrate:    getKubernetesNodePool,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the node pool.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "selfLink",
+				Description: "Server-defined URL for the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "location_type",
+				Description: "Location type of the cluster i.e REGIONAL/ZONAL.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.From(gcpKubernetesNodePoolLocationType),
+			},
+			{
+				Name:        "status",
+				Description: "The status of the nodes in this pool instance.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cluster_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "Cluster in which the Node pool is located.",
+				Transform:   transform.FromP(kubernetesNodePoolTurbotData, "ClusterName"),
+			},
+			{
+				Name:        "initial_node_count",
+				Description: "The initial node count for the pool.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "version",
+				Description: "The version of the Kubernetes of this node.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "locations",
+				Description: "The initial node count for the pool.",
+				Type:        proto.ColumnType_STRING,
+			},
```

Comment:
Fix column type, and add proper description
https://github.com/googleapis/google-api-go-client/blob/master/container/v1/container-gen.go#L3312-L3319